base/replutil.jl

```diff
@@ -414,8 +414,14 @@ function showerror_ambiguous(io::IO, meth, f, args)
         i < length(p) && print(io, ", ")
     end
     print(io, ") is ambiguous. Candidates:")
+    sigfix = Any
     for m in meth
         print(io, "\n  ", m)
+        sigfix = typeintersect(m.sig, sigfix)
+    end
+    if isa(unwrap_unionall(sigfix), DataType) && sigfix <: Tuple
+        print(io, "\nPossible fix: ")
```

Comment:
How would you word that into a sentence?
"Possible fix, define f(::Integer, ::Integer)"
"Intersection of ambiguous definitions: f(::Integer, ::Integer)"
"Missing candidate definition: f(::Integer, ::Integer)"